Source/SwiftLintFramework/Rules/FirebaseConfigActivateRule.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  FirebaseConfigActivateRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigActivateRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_activate",
+        name: "Firebase Config Activate",
+        description: "Firebase Config should be activated.",
+        nonTriggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n self.remoteConfig.activateFetched() \n }"
+        ],
+        triggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n }"
+        ]
+    )
+
+    public func validate(file: File, kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard SwiftExpressionKind.call == kind else {
+            return []
+        }
+
+        guard dictionary.name!.hasSuffix(".fetch") else {
+            return []
+        }
+
+        guard dictionary.substructure[0].name == "withExpirationDuration" else {
```

Comment:
The same logic applies here. if a method called `.fetch` (that's not the Firebase `.fetch`) doesn't take any parameter, it won't have a substructure in `[0]` (or `.first`).